include/openssl/ct.h

```diff
@@ -347,14 +350,14 @@ void SCT_LIST_print(const STACK_OF(SCT) *sct_list, BIO *out, int indent,
  * Returns 1 if the SCT verifies successfully, 0 if it cannot be verified and a
  * negative integer if an error occurs.
  */
-int SCT_verify(const SCT_CTX *sctx, const SCT *sct);
+__owur int SCT_verify(const SCT_CTX *sctx, const SCT *sct);
 
 /*
  * Verifies an SCT against the provided data.
  * Returns 1 if the SCT verifies successfully, 0 if it cannot be verified and a
  * negative integer if an error occurs.
  */
-int SCT_verify_v1(SCT *sct, X509 *cert, X509 *preissuer,
+__owur int SCT_verify_v1(SCT *sct, X509 *cert, X509 *preissuer,
```

Comment:
As above, X509 functions aren't all constified so we need non-const parameters here for compatibility.